### PR TITLE
actually pass the network used through to solitude (bug 926575)

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

**Comment by kumar303** [comment]
looks like legit test failures

File: lib/solitude/api.py [hunk 1]
```diff
@@ -592,8 +592,8 @@ def create_transaction(self, generic_buyer, generic_seller,
         # In the case of Zippy, it does not detect any of these values
         # itself. All other providers will detect these values without
         # help from Webpay.
-        carrier = 'USA_TMOBILE'
-        region = '123'
+        carrier = 'FAKE'
+        region = 'CAN'
```
Comment (kumar303) [inline lib/solitude/api.py]: I thought region was the mcc, no?

**Comment by kumar303** [comment]
r+wc after test updates

File: lib/solitude/api.py (hunk 1 above)
Comment (andymckay) [inline lib/solitude/api.py]: We actually get back from bango a network and region like USA_TMOBILE. in solitude its just a charfield. As long as we get back something that conforms to ISO 3166-1 and can easily be run through mobile-codes, I'm happy :)

Also long term I don't know if mcc makes sense for non-SIM enabled devices.

**Pull request merged**